### DISCUSSION TITLE
A form handle is required on Form tags error while caching

### DISCUSSION THREAD

**Discussion by pascallieverse** [statamic/cms#7621] [Troubleshooting]
Currently we are experiencing issues that sometimes after deploying and refreshing the static and stache cache some pages with forms give a 500 error with the following message, meanwhile other pages with forms are working normally.

`A form handle is required on Form tags. Please refer to the docs for more information.`

The stache watcher is turned off and the caching strategy is tried on half and full, the forms are loaded with a nocache as seen in the example below, but the error keeps coming back on different pages sometimes and are gone after refreshing the cache.

```
        {{ nocache }}
        {{ form:create
            in=form
            id="form-{form}"
            csrf="false"
            x-ref="form"
            @submit.prevent="sendForm()"
            js="alpine:dynamic_form"
        }}
```

Anyone got an idea how to fix this problem from coming back?

**Comment by duncanmcclean** [comment]
You need to fix the syntax for the `in` parameter it should be like this...

```antlers
{{ form:create
            :in="form"
            id="form-{form}"
            csrf="false"
            x-ref="form"
            @submit.prevent="sendForm()"
            js="alpine:dynamic_form"
}}
```

Does that help at all?

**Reply by jasonvarga** [reply]
@pascallieverse Just to clarify, Duncan is saying you need to make sure there are quotes around the `in` parameter. And if you're intending to pass the `form` _variable_, and not just the literal string "form" - then you also need the `:` prefix.

```diff
-in=form
+:in="form"
```

**Reply by pascallieverse** [reply]
Hi @jasonvarga Yes I noticed that after my comment, I just did the change a few minutes ago but the error is still coming up.

**Reply by RyanRoberts** [reply]
> Hi @jasonvarga Yes I noticed that after my comment, I just did the change a few minutes ago but the error is still coming up.

This is a long shot since you originally posted back in 2023, but do you have any recollection of whether you resolved this issue or not? Because somehow I am coming upon the same problem.
